src/common/utils/validateDuplicateFieldValue.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { validateRequired } from '@folio/stripes-acq-components';
+
+export const validateDuplicateFieldValue = async ({
+  ky,
+  api,
+  id,
+  fieldValue,
+  errorMessage,
+  fieldName,
+  query = `${fieldName} == "${fieldValue}"`,
```

Comment:
it there a case when we don't filter records like this?

Reply:
yes, for searching fund name https://github.com/folio-org/ui-finance/pull/512/files#diff-1c86fe653eb3a04d2f74a277716a5c0e4ba5d62ef829f71c84ceea2176596d22R80